cloud/workspace/workspace_token.go

```diff
@@ -205,13 +214,21 @@ func RotateToken(id, name, workspace string, cleanOutput, force bool, out io.Wri
 	if workspace == "" {
 		workspace = ctx.Workspace
 	}
-	tokens, err := getWorkspaceTokens(workspace, client)
-	if err != nil {
-		return err
-	}
-	token, err := getWorkspaceToken(id, name, workspace, "\nPlease select the Workspace API token you would like to rotate:", tokens)
-	if err != nil {
-		return err
+	var token astrocore.ApiToken
```

Comment:
refactor that reduces load on api server. If we have the id use the id instead of getting a list of all org tokens.